web/templates/invite_user_modal.hbs

```diff
@@ -32,6 +32,13 @@
             <textarea rows="2" id="invitee_emails" name="invitee_emails" class="invitee_emails" placeholder="{{t 'One or more email addresses...' }}"></textarea>
         </div>
     </div>
+    <div class="input-group new-style" id="receive-invite-acceptance-notification">
```

Comment:
Ohh I though this ID was used for the checkbox input element. Would probably be better to use this ID there and here you can either add `_container` at the end or just use `$("#receive-invite-acceptance-notification").closest(".input-group")` to access this in the JS code.